src/mscorlib/corefx/System/Globalization/TextInfo.cs

```diff
@@ -38,38 +36,28 @@ public partial class TextInfo : ICloneable, IDeserializationCallback
         ////
 
         [OptionalField(VersionAdded = 2)]
-        private String m_listSeparator;
```

Comment:
This is the case for much, much more than this one case.  Every field of every type in corefx that's serializable (or that will be soon) likely suffers from this.  We've put a stake in the ground that serialization will not be supported across platforms, e.g. you can't serialize on one platform and deserialize on another.  If we have to revisit that, we're going to need to do a ton of work to fix up all such cases.